src/test/scala/com/gu/identity/api/IdentityApiSanityTest.scala

```diff
@@ -0,0 +1,42 @@
+package com.gu.identity.api
+
+import com.gu.util.TargetHost
+import com.gu.util.Http
+
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers
+import scala.util.Success
+import scala.util.Failure
+import org.slf4j.LoggerFactory
+
+/**
+ * Sanity test of public Identity API on CODE
+ */
+class IdentityApiSanityTest extends FlatSpec with Matchers {
+
+  private[this] val logger = LoggerFactory.getLogger(this.getClass)
+
+  private[this] val idApiHost = TargetHost.idApiHost() match {
```

Comment:
This logic should be moved the the TargetHost class and executed on initialisation.